core/src/main/java/com/sequenceiq/cloudbreak/repository/StackUpdater.java

```diff
@@ -86,6 +86,7 @@ private Stack doUpdateStackStatusReason(Long stackId, String statusReason) {
         Stack stack = stackRepository.findById(stackId);
         if (statusReason != null) {
             stack.setStatusReason(statusReason);
+            cloudbreakEventService.fireCloudbreakEvent(stackId, stack.getStatus().name(), statusReason);
```

Comment:
Have you noticed that this change will raise a notification to the UI too in every 5 seconds while Azure copying the image?